.circleci/config.yml

```diff
@@ -108,7 +109,15 @@ jobs:
           working_directory: /home/circleci
           command: |
             echo "Master theme branch is ${CIRCLE_BRANCH}"
-            MASTER_THEME_BRANCH=dev-${CIRCLE_BRANCH} MERGE_SOURCE=git@github.com:greenpeace/planet4-base-fork.git MERGE_REF=develop make
+            if [ -n "$CIRCLE_PR_NUMBER" ]; then
+              BRANCH="$(./get-fork-pr-branch.sh)"
+            else
+              BRANCH="${CIRCLE_BRANCH}"
+            fi
+            MASTER_THEME_BRANCH=dev-${BRANCH}#${CIRCLE_SHA1} \
```

Comment:
I added the `$CIRCLE_SHA1` to make the pipeline use the exact commit for which it started, regardless of what the head of the branch is. I don't remember if I it was actually required to make it work, but I guess it won't hurt?